<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StimConfig</class>
 <widget class="QDockWidget" name="StimConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>941</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>325</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stimulation Configuration</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="3">
     <widget class="QCheckBox" name="pchgModCheck">
      <property name="text">
       <string>Use diode offset</string>
      </property>
     </widget>
    </item>
    <item row="4" column="12">
     <widget class="QComboBox" name="clBox"/>
    </item>
    <item row="8" column="11">
     <widget class="QLabel" name="stimBiasLabel">
      <property name="text">
       <string>Stim. Bias</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QCheckBox" name="blkAllCheck">
      <property name="text">
       <string>Block All</string>
      </property>
     </widget>
    </item>
    <item row="6" column="11">
     <widget class="QLabel" name="stimFrequencyLabel">
      <property name="text">
       <string>Stim. Period</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QCheckBox" name="testEnCheck">
      <property name="text">
       <string>Test enable</string>
      </property>
     </widget>
    </item>
    <item row="11" column="6">
     <widget class="QCheckBox" name="testSelCheck">
      <property name="toolTip">
       <string>T: compliance voltage; 0: mirror headroom</string>
      </property>
      <property name="text">
       <string>Test select</string>
      </property>
     </widget>
    </item>
    <item row="5" column="11">
     <widget class="QLabel" name="nccpLabel">
      <property name="text">
       <string># of comp. pulses</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="12">
     <widget class="QDoubleSpinBox" name="nccpBox">
      <property name="suffix">
       <string/>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>63.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="11">
     <widget class="QLabel" name="stimMultLabel">
      <property name="text">
       <string>Stim. Multiplier</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="11">
     <widget class="QLabel" name="clLabel">
      <property name="text">
       <string>Compliance Level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="12">
     <widget class="RoundedSpinBox" name="stimBiasBox">
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
      </property>
      <property name="suffix">
       <string> nA</string>
      </property>
      <property name="decimals">
       <number>0</number>
      </property>
      <property name="minimum">
       <double>45.000000000000000</double>
      </property>
      <property name="maximum">
       <double>150.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>15.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="11">
     <widget class="QLabel" name="hrcLabel">
      <property name="text">
       <string>Headroom Control</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="12">
     <widget class="QComboBox" name="stimMultBox"/>
    </item>
    <item row="3" column="12">
     <widget class="QComboBox" name="hrcBox"/>
    </item>
    <item row="9" column="6">
     <widget class="QCheckBox" name="prCheck">
      <property name="toolTip">
       <string>T: monophasic; F: biphasic</string>
      </property>
      <property name="text">
       <string>Monophasic</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="gndResCheck">
      <property name="text">
       <string>Bypass ref R</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="refPdCheck">
      <property name="text">
       <string>Ref to 10M R</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QCheckBox" name="pkgRetCheck">
      <property name="text">
       <string>Package return</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QCheckBox" name="pkgPdCheck">
      <property name="text">
       <string>Bypass pkg R</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QCheckBox" name="cgndCheck">
      <property name="text">
       <string>GND Counter</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QCheckBox" name="polCheck">
      <property name="text">
       <string>Bipolar</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QCheckBox" name="regBiasCheck">
      <property name="text">
       <string>Use external bias</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QCheckBox" name="gmBiasCheck">
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>Use constant-gm</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QCheckBox" name="gmPulseCheck">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Force gm startup</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QCheckBox" name="noCheckCheck">
      <property name="text">
       <string>Disable error check</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QCheckBox" name="stopModeCheck">
      <property name="text">
       <string>Short during e-stop</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QCheckBox" name="doffCheck">
      <property name="toolTip">
       <string>T: one diode; F: two diodes</string>
      </property>
      <property name="text">
       <string>Use 1 diode</string>
      </property>
     </widget>
    </item>
    <item row="6" column="12">
     <widget class="QDoubleSpinBox" name="stimFrequencyBox">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="prefix">
       <string>value Hz (</string>
      </property>
      <property name="suffix">
       <string> ms)</string>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>3.910000000000000</double>
      </property>
      <property name="maximum">
       <double>67.843800000000002</double>
      </property>
      <property name="singleStep">
       <double>0.062500000000000</double>
      </property>
      <property name="value">
       <double>3.910000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="12">
     <widget class="QPushButton" name="writeRegs">
      <property name="text">
       <string>Write Stim. Registers</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="6">
     <widget class="QPushButton" name="waveformConfigButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Waveform Configuration</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RoundedSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>roundedspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
